sdks/java/io/cdap/src/test/java/org.apache.beam.sdk.io.cdap/ConfigWrapperTest.java

```diff
@@ -53,27 +50,30 @@
           .put("loginUrl", "https://www.google.com")
           .put("referenceName", "oldReference")
           .build();
-  private static final String TEST_SALESFORCE_PARAMS_JSON_STRING = "{\n" +
-      "\"sObjectName\": \"sObject\",\n" +
-      "\"datetimeAfter\": \"datetime\",\n" +
-      "\"consumerKey\": \"key\",\n" +
-      "\"consumerSecret\": \"secret\",\n" +
-      "\"username\": \"user\",\n" +
-      "\"password\": \"password\",\n" +
-      "\"loginUrl\": \"https://www.google.com\",\n" +
-      "\"referenceName\": \"reference\"\n" +
-      "}";
-  private static final String SALESFORCE_PARAMS_JSON_FILE_PATH_PROPERTY = "salesforceParamsJsonFilePath";
+  private static final String TEST_SALESFORCE_PARAMS_JSON_STRING =
+      "{\n"
+          + "\"sObjectName\": \"sObject\",\n"
+          + "\"datetimeAfter\": \"datetime\",\n"
+          + "\"consumerKey\": \"key\",\n"
+          + "\"consumerSecret\": \"secret\",\n"
+          + "\"username\": \"user\",\n"
+          + "\"password\": \"password\",\n"
+          + "\"loginUrl\": \"https://www.google.com\",\n"
+          + "\"referenceName\": \"reference\"\n"
+          + "}";
+  private static final String SALESFORCE_PARAMS_JSON_FILE_PATH_PROPERTY =
+      "salesforceParamsJsonFilePath";
   public static final String REFERENCE_NAME_PARAM_NAME = "referenceName";
 
   @Test
   public void testBuildingPluginConfigFromParamsMap() {
     try {
       String newReferenceName = "new reference name";
-      SalesforceSourceConfig firstConfig = new ConfigWrapper<>(SalesforceSourceConfig.class)
-          .withParams(TEST_SALESFORCE_PARAMS_MAP)
-          .setParam("referenceName", newReferenceName)
-          .build();
+      SalesforceSourceConfig firstConfig =
+          new ConfigWrapper<>(SalesforceSourceConfig.class)
+              .withParams(TEST_SALESFORCE_PARAMS_MAP)
+              .setParam("referenceName", newReferenceName)
```

Comment:
I guess this variable may be used here.

```suggestion
              .setParam(REFERENCE_NAME_PARAM_NAME, newReferenceName)
```

Reply:
Done